statsmodels/graphics/tsaplots.py

```diff
@@ -23,19 +23,22 @@ def _prepare_data_corr_plot(x, lags, zero):
     return lags, nlags, irregular
 
 
-def _plot_corr(ax, title, acf_x, confint, lags, irregular, use_vlines, **kwargs):
+def _plot_corr(ax, title, acf_x, confint, lags, irregular, use_vlines,
+               vlines_kwargs, **kwargs):
     if irregular:
         acf_x = acf_x[lags]
         if confint is not None:
             confint = confint[lags]
 
     if use_vlines:
-        ax.vlines(lags, [0], acf_x, **kwargs)
+        ax.vlines(lags, [0], acf_x, **vlines_kwargs)
         ax.axhline(**kwargs)
 
     kwargs.setdefault('marker', 'o')
     kwargs.setdefault('markersize', 5)
-    kwargs.setdefault('linestyle', 'None')
+    if 'ls' not in kwargs:
```

Comment:
users could also use the long form 'linestyle'
`if 'ls' not in kwargs and 'linestyle' not in kwargs`

Reply:
This isn't needed since the next line is `setdefault` which won't overwrite an existing 'linestyle'

Reply:
Good, I didn't realize that.

Reply:
Bad MPL design unfortunately.